service_info_cms/views.py

```diff
@@ -0,0 +1,17 @@
+from cms.models.pagemodel import Page
+from django.shortcuts import redirect
+from .models import PageRating
+
+
+def update_page_rating(request):
```

Comment:
can you add a simple test of this with/without valid params?  look in service_info_cms.utils.create_essential_pages() to get a hint on creating a CMS page for the testcase